tests/lwip/main.c

```diff
@@ -39,7 +39,7 @@ static int ifconfig(int argc, char **argv)
     (void)argc;
     (void)argv;
     for (struct netif *iface = netif_list; iface != NULL; iface = iface->next) {
-        printf("%s_%02u: ", iface->name, iface->num);
+        printf("%c%c_%02u: ", iface->name[0], iface->name[1], iface->num);
```

Comment:
Why this change btw?

Reply:
Because `netif.name` is defined as `char name [2]` and set to `E` + `T` here https://github.com/RIOT-OS/RIOT/blob/ae6e48cd677bfc49df3b6058a3f805ace3069bcf/pkg/lwip/contrib/netdev/lwip_netdev.c#L111-L112 There is no terminating `\0`. So can't be printed as string with `%s`. With `%s` you get garbage if the `netif.num` (the subsequent element) is not 0. It works for netif 0 but not for netifs > 0. That's probably why we didn't realize that.